packages/components/src/common/toggler/index.js

```diff
@@ -0,0 +1,3 @@
+import Toggler from './Toggler';
+
+export default Toggler;
```

Comment:
```suggestion
export { default } from './Toggler';
```

please make this a `.ts` file as well